sample/01-all-features/src/health/health.interface.ts

```diff
@@ -0,0 +1,19 @@
+export interface DatabaseConnectionDetails {
```

Comment:
can we rename the class to be Postgres specific? 
If this same structure can be used for other SQL based DBs? In that case can we turn this one into a generic object?